spec.md

```diff
@@ -130,16 +130,18 @@ Unlisted groups MAY be masks.
 
 ### "color"
 
-The `color` key defines an image that is "labeled", i.e. every unique value in the image
+The `color` key defines an integer image that is "labeled", i.e. every unique non-negative value in the image
 represents a unique, non-overlapping object within the image. The value associated with
-the `color` key is another JSON object in which the key is the pixel value of the image and
-the value is an RGBA color (4 byte, `0-255` per channel) for representing the object:
+the `color` key is a JSON array in which the index is the pixel value of the image and
+the value is an RGBA color (4 byte, `0-255` per channel) for representing the object, or `null` if no color is set:
```

Comment:
We should say something about how the 4-bytes are converted to a single int, e.g. signed/unsigned, big/little endian